assets/dev/scss/frontend/widgets/divider.scss

```diff
@@ -137,11 +137,28 @@
 	direction: rtl;
 }
 
-.e-container {
+.elementor {
 
-	& > .elementor-widget-divider {
-		// Fix - Divider widget has undefined width and height issues inside the flex container (ED-7722).
-		width: var( --container-widget-width, 100% );
-		flex-grow: 1;
+	.elementor-element {
+
+		&.elementor-section {
+
+			.elementor-divider-separator {
+				width: var( --divider-width, 100% );
```

Comment:
This is a really small but best we keep with the standard in mostly all of the El `.scss`'s
wrong: `var( --divider-width, 100% );`
right: `var(--divider-width, 100%);`

Can you chnage this for all the `var()`'s

Small one, but thanks

Reply:
@StuartElementor 
I have changed it. 

I have checked it with Udi in the passed, but I believe he indicated that both versions are allowed inside scss. There isn't clarity about this.